### DISCUSSION TITLE
How to click an image to go to a link

### DISCUSSION THREAD

**Discussion by Sillybutts** [mmistakes/minimal-mistakes#3665] [Q&A] [answered]
Sorry for the EXTREMELY basic question, but I've tried all the limited markdown knowledge I have, and the several threads I've found have been unhelpful. I am _not_ a regular user of GitHub, so I'm not very familiar with how Pages mess with how things "normally work".

I just want to place an image on a page, where you click on the image to be taken to a link. So I can click on a picture of a cat and it would take me to the adoption page for a shelter, for example.

**Comment by mmistakes** [comment, marked as answer]
https://talk.jekyllrb.com/t/how-to-add-a-image-with-links-in-markdown/5915/2

**Comment by Sillybutts** [comment]
Thank you very much, the `[![FLipkart](/assets/img/buttons/flipkart.png)](https://www.flipkart.com/){:target="_blank"}` worked exactly as I wanted!

Is it possible to make the [feature row](https://mmistakes.github.io/minimal-mistakes/docs/helpers/#feature-row) images work the same way? I don't see anything in the documentation that mentions making images act as link openers.

**Reply by mmistakes** [reply]
Not without customization. The buttons in a feature row item are the only thing currently linkable. Here are the relevant lines if you want to look them over and adjust.

https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/feature_row#L33-L35

**Reply by Sillybutts** [reply]
Gotcha, thank you. I think I'll just use plain markdown in that case, foregoing the feature row.
